Front-end/src/components/Post.jsx

```diff
@@ -171,7 +171,7 @@ const Post = (props) => {
           </p>
         </div>
         <h1 className="title">{props.title}</h1>
-        <h3 className="description">
+        <h3 className="p-description">
```

Comment:
QQ - what's the difference?

Reply:
p-description is to specify it was for posts. I had a css conflict with another element that had the same class name as "description"